folium/features.py

```diff
@@ -232,7 +232,7 @@ class VegaLite(Element):
 
     def __init__(self, data, width=None, height=None,
                  left='0%', top='0%', position='relative'):
-        super(VegaLite, self).__init__()
+        super(self.__class__, self).__init__()
```

Comment:
Please keep this PR clean and don't change this here.

Reply:
I somehow got a strange error at some point, when I used `VegaLite` instead of `self.__class__`, but I can revert that.

Reply:
What was the error?